x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/vulnerabilities_finding_flyout/vulnerability_overview_tab.tsx

```diff
@@ -51,12 +62,26 @@ const CVSScore = ({ vectorBaseScore, vendor }: CVSScoreProps) => {
       <EuiFlexItem
         grow={false}
         css={css`
-          width: 94px;
+          width: 116px;
           font-weight: 600;
+          display: flex;
+          flex-direction: row;
+          align-items: center;
         `}
       >
-        {vendorName}
+        <span
+          css={css`
+            margin-right: 30px;
```

Comment:
why do we need margin right on both this element and directly on the icon? maybe just one would suffice

Reply:
I'll remove margin from the logo.